tests/src/test/scala/org/http4s/headers/RefererSuite.scala

```diff
@@ -22,7 +22,7 @@ import org.http4s.syntax.header._
 import org.http4s.laws.discipline.ArbitraryInstances._
 
 class RefererSuite extends HeaderLaws {
-  //checkAll("Referer", headerLaws(`Retry-After`))
+  checkAll("Referer", headerLaws(`Retry-After`))
```

Comment:
```suggestion
  checkAll("Referer", headerLaws[`Retry-After`])
```

Reply:
Ugh, yes.  This was an in-browser drive-by.